src/org.xtuml.bp.ui.properties/arc/BuildPropertySource.arc

```diff
@@ -757,7 +757,8 @@ val = val${count};
               Attribute_c base_attr = Attribute_c.getOneO_ATTROnR106(BaseAttribute_c.getOneO_BATTROnR113(rattr));
               if ( (  (base_attr == null)) )
               {
-                return "same_as<Base_Attribute>";
```

Comment:
Updated the int. Let me know if it is still unclear. For convenience, here is the section copied from the note:
The change in `BuildPropertySource.arc` was introduced to allow the properties
view to correctly display types for baseless referential attributes. Before the
change, the property source did check if a base attribute across R113 is null,
but in the case where no base attribute was found it would simply display the
hardcoded value "same_as<Base_Attribute>". This is bad in many ways, but most
relevantly, it obfuscates any data type that is related to a baseless
referential attribute.  The code has been changed to instead display the name of
the data type associated with the referential attribute in the case of no base
attribute being found across R113.